olive/passes/onnx/perf_tuning.py

```diff
@@ -280,6 +292,11 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
                 default_value=False,
                 description="Whether enable IOBinding Search for ONNX Runtime inference.",
             ),
+            "enable_cuda_graph": PassConfigParam(
```

Comment:
Will we automatically use `CUDAExecutionProvider` if `enable_cuda_graph` was true even if no `CUDAExecutionProvider` in `providers_list`?

Reply:
We will use `CUDAExecutionProvider` only if `CUDAExecutionProvider` is specified, or if it is in available providers list if user doesn't specify the `providers_list`. It would be better to keep this logic.